Modules/core/View/Corespaceuser/editAction.twig

```diff
@@ -0,0 +1,195 @@
+                        let dynamicForms = new DynamicForms();
+                        let spaceId = {{ id_space }};
+                        let sourceId = "resource";
+                        let targets = [
+                            {
+                                elementId: "visa_id",
+                                apiRoute: `revisas/getcategoryvisas/`,
+                                activateOnLoad: true
+                            }
+                        ];
+                        dynamicForms.dynamicFields(sourceId, targets, spaceId);
+                        app.$data.dynamicsLoaded = true;
+                    }
+                </script>
+
+                <script>
+                    function ConfirmDelete(id, name) {
+                        let spaceId = {{ id_space }}
+                        let userId = {{ id_user }}
+                        if (confirm(`Delete ${name} ?`)) {
+                            location.href=`clientsuseraccountsdelete/${spaceId}/${userId}/${id}`;
+                        }
+                    }
+                </script>
+
+            </div>
+        </div>
+
+    </div>
+</div>
+
+<script>
+    class Section {
+        constructor(name, module, btnId, displayName, formHtml = null, tableHtml = null, show = false, subSection = null) {
+            this.name = name;
+            this.module = module;
+            this.btnId = btnId;
+            this.displayName = displayName;
+            this.formHtml = formHtml;
+            this.tableHtml = tableHtml;
+            this.subSection = subSection;
+            this.show = show;
+        }
+    }
+    var app = new Vue({
+        el: '#formSection',
+        data () {
+            return {
+                origin: false,
+                options: [],
+                forms: [],
+                sections: [],
+                params: {
+                    spaceId: {{ id_space }},
+                    userId: {{ id_user }},
+                    lang: 'en'
+                },
+                dynamicsLoaded: false
+            }
+        },
+        mounted: function() {
+            this.origin = JSON.parse(this.$el.attributes['data-origin'].value);
+            this.options = JSON.parse(this.$el.attributes['data-options'].value);
+            this.forms = JSON.parse(this.$el.attributes['data-forms'].value);
+            this.lang = JSON.parse(this.$el.attributes['data-lang'].value);
+            this.generateSections();
+            this.showForm(this.getOriginPage());
+        },
+        methods: {
+            showForm(itemModule) {
+                let found = false;
+                this.sections.forEach((section, index, array) => {
+                    if (section.module === itemModule) {
+                        section.show = true;
+                        found = true;
+                    } else {
+                        if (section.subSection && section.subSection.module === itemModule) {
```

Comment:
why hiding booking subsection (auth history)?

Reply:
It seemed to add too much informations when in most cases admins just want to add or check authorisations

Reply:
it is just a table below

Reply:
true. Because I made a lot of tests, it is a big table, but you're right, it's not so big normally. I'll make it show and remove the show/hide button